docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/IndicateMissingValues.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class IndicateMissingValues
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create a TextLoader for the breast-cancer dataset, and load it into an IDataView.
+            var loader = mlContext.Data.CreateTextLoader(new[]
+            {
+                new TextLoader.Column("Label", DataKind.Single, 0),
+                new TextLoader.Column("Features1", DataKind.Single, 1, 5),
+                new TextLoader.Column("Features2", DataKind.Single, 7, 9),
+                new TextLoader.Column("Missing", DataKind.Single, 6)
+            });
+            var file = SamplesUtils.DatasetUtils.DownloadBreastCancerDataset();
+            var data = loader.Load(new MultiFileSource(file));
```

Comment:
i think we want to move forward with just a small in-memory IEnumerable for the data, and skip the TextLoader and the real datasets.

Reply:
you can find a sample with small in-memory dataset in this comments: [https://github.com/dotnet/machinelearning/issues/2726#issuecomment-468781807](https://github.com/dotnet/machinelearning/issues/2726#issuecomment-468781807)

---
In reply to: [261707265](https://github.com/dotnet/machinelearning/pull/2814#discussion_r261707265) [](ancestors = 261707265)